modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/H2SchemaManager.java

```diff
@@ -221,6 +222,12 @@ private void dropTable(H2TableDescriptor tbl) {
 
     /** {@inheritDoc} */
     @Override public void onFunctionCreated(String schema, String name, boolean deterministic, Method method) {
+        if (!Modifier.isStatic(method.getModifiers())) {
+            log.warning("Skip creating SQL function '" + name + "' in H2 engine because it is not static.");
```

Comment:
Let's use `LT.warn` here to reduce number of logging in case user application generates a lot of query with the same UDF.

Reply:
This log message appears only once, when cache is created.